jans-cedarling/cedarling/src/init/policy.rs

```diff
@@ -0,0 +1,139 @@
+/*
+ * This software is available under the Apache-2.0 license.
+ * See https://www.apache.org/licenses/LICENSE-2.0.txt for full text.
+ *
+ * Copyright (c) 2024, Gluu, Inc.
+ */
+
+use std::collections::HashMap;
+
+use base64::prelude::*;
+use cedar_policy::PolicyId;
+
+// we use camel case to show that it is like a constant
+#[allow(non_camel_case_types)]
+#[derive(Debug, thiserror::Error)]
+enum ParsePolicySetErrMsg {
```

Comment:
`ParsePolicySetError`

Reply:
Discussed that it is not error, but list of the messages.
Open to ideas how to make it more descriptive

Reply:
Renamed to `ParsePolicySetMessage`